content/1.7/back-office/shop-parameters/contact/add-edit-contacts.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Add / edit contact
+weight: 1
+---
+# Add / edit contact
+
+![Contact](/img/contacts.png)
+
+Click the "Add new contact" button or edit an existingg contact to access the contact creation form.
+
+While adding or editing a contact, the following fields / toggles / parameters are displayed:
+
+ 1) Title
+
+This field is mandatory and can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+Title is the name of the contact: either a person's name or the name of the service itself. Has an explanation below the field written: "Contact name (e.g. Customer Support)."
```

Comment:
```suggestion
The title is the name of the contact. For example, it can be either a person's name or the name of the service itself. There is a help-text under this field: "Contact name (e.g. Customer Support)."
```